### PR TITLE
Draft of the new /embedded page

### LINKED ISSUE #4864: /embedded page [build]

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Created a quick demo of the /embedded page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/embedded](http://0.0.0.0:8001/embedded)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare to the [copy doc](https://docs.google.com/document/d/15Tz87YRg4SnX4DWGOSl2TDTHLSStJoma879xHnF0nuo/edit#heading=h.c6wjbvvp8mcm)

## Issue / Card

Fixes #4864

### CODE REVIEW

**Comment by anasereijo** [comment]
Quick question that will help me to review everything:

I'm slightly confused with the hierarchy of this page.
Some sections have "heroes banners" (light grey with _suru_) and other sections don't. Which makes me not understanding if I click on one item from the sticky nav if it will be a new section or just a sub-section? It's quite hard to explain, sorry.

I opened these 2 sections from links on the sticky nav. They look different hierarchy wise. Should this be like the Core page were you can clearly see a hirarchy of sections and sub sections?

Screenshot 2019-03-28 at 12 05 28

Screenshot 2019-03-28 at 12 05 46

**Comment by anasereijo** [comment]
Update:

@pmahnke and I went through the copydoc and rearrange things in order to have more a sense of hierarchy. We are waiting from Erics feedback.

**Comment by anasereijo** [comment]
@pmahnke and I went through the [page](https://docs.google.com/document/d/1MLYiCFXVLkDIEp4E9_CmrNmt8n0oAd6qcxmcf-9XcnY/edit?ts=5c91071c#heading=h.mg9r4i2ak8cb) and reorganised it in sections. We believe this [new version](https://docs.google.com/document/d/15Tz87YRg4SnX4DWGOSl2TDTHLSStJoma879xHnF0nuo/edit#heading=h.c6wjbvvp8mcm) has a clearer sense of hirearchy making it easier to read. We also have new illustrations. 

- [ ] waiting for Eric's feedback on the new page and illustrations

**Comment by anasereijo** [comment]
Eric is happy with the new page layout. I left some questions regarding links, hopefully we'll have those very soon.

@anthonydillon you can start building this as all assets are saved down for you.

**Comment by anasereijo** [comment]
- [ ] please make sure all images are `.u-vertically-center`

- [ ] don't resize the images. we saved them all with the same size to be proportional. you don't need `width:` (except canonical logo - that's ok as it is now)

- [ ] Docker logo seems smaller than the other ones. please amend (let me know if you need my help)

- [ ] 'All kinds of containers' section can we have links instead of neutral buttons? Looks very boxy and white at the moment.

- [ ] can we have our list style in here? the ticked bullets look nicer
Screenshot 2019-04-03 at 17 48 15

- [ ] make the text `h4` as other sections
Screenshot 2019-04-03 at 17 49 18

- [ ] remove the following `hr`. I think these 2 sub sections shouldn't be seperated 

Ubuntu is the new standard for embedded Linux   Ubuntu

**Comment by anthonydillon** [comment]
@anasereijo addressed all your comments. The docker SVG has a lot of white space around it. Can you remove it please?

**Comment by anasereijo** [comment]
@anthonydillon Eric just updated the doc. It is mainly text changes. I think you should make those changes first and I review it afterwards. In the meantime I will provide you a new Docker logo.

**Comment by anasereijo** [comment]
@anthonydillon docker logo: https://assets.ubuntu.com/v1/04100753-2018-logo-docker.svg

**Comment by anasereijo** [comment]
- [ ] I know it's my fault but I think it doesn't look good when we have the text on the right and image on the left. Can you please amend that? (except on "Faster, cheaper, better" - this one looks good as it is)

- [ ] "Classic Ubuntu Server or new Ubuntu Core" section: fix spacing

Ubuntu is the new standard for embedded Linux   Ubuntu (1)

- [ ] please replace docker logo (it's attached to a comment I made earlier)

**Comment by anthonydillon** [comment]
@anasereijo all done, ready for you again.

**Comment by anasereijo** [comment]
- [ ] Eric mentioned he made one small wording change in the [copydoc](https://docs.google.com/document/d/15Tz87YRg4SnX4DWGOSl2TDTHLSStJoma879xHnF0nuo/edit#heading=h.c6wjbvvp8mcm)

- [ ] Docker logo should be left aligned 
Screenshot 2019-04-08 at 09 36 37

- [ ] 'Hardware? Easy.' header: can each sentence fit on one line?

**Comment by anthonydillon** [comment]
@anasereijo All done, the docker SVG has some styling in it that centers the logo only on Chrome. Is there a different asset we could use?

**Comment by anasereijo** [comment]
@anthonydillon new docker logo https://assets.ubuntu.com/v1/417fe47d-2018-logo-docker-01.svg

**Comment by anthonydillon** [comment]
@anasereijo done

**Comment by anasereijo** [comment]
@anthonydillon docker logo looks the same

**Comment by anasereijo** [comment]
@anthonydillon docker logo looks the same to me.
there is also a comment from Eric on the copydoc then I think it's good to go!

**Comment by anasereijo** [comment]
I've pinged you the docker logo hope it works now

sorry just a final round of amends and I guess it's ready
Ubuntu is the new standard for embedded Linux   Ubuntu

**Comment by anthonydillon** [comment]
@anasereijo addressed your points. The docker image has whitespace around it.

**Comment by anthonydillon** [comment]
@anasereijo that docker image worked :tada:

**Comment by anasereijo** [comment]
@anthonydillon looking good 👍 but once again the docker logo isn't displaying properly. can you try the png ?

File: templates/embedded/index.html [hunk 1]
```diff
@@ -0,0 +1,694 @@
...
+
+<section class="p-strip">
+  <div class="row p-divider">
+    <div class="col-6 p-divider__block">
+      <h4>Reference platforms</h4>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Raspberry Pi</li>
+        <li class="p-list__item is-ticked">Intel NUC</li>
+        <li class="p-list__item is-ticked">Qualcomm Dragonboard</li>
+        <li class="p-list__item is-ticked">i.MX6 SABRE Lite board</li>
```
Comment (steverydz) [inline templates/embedded/index.html]: The copy doc has (coming soon) after this one

**Pull request merged**